doc/commands/repoquery.8.rst

```diff
@@ -78,6 +78,11 @@ Options
 ``--installed``
     | Display only installed packages.
 
+``--userinstalled``
+    | Limit to packages that are not installed as dependencies or weak dependencies.
+    | This means limit to packages that were installed at the user request or indirectly as a part of a module profile or comps group. Additionally it returns packages with unknown reason.
```

Comment:
I am thinking about modiffication of - `Additionally it returns packages with unknown reason.`
What about ``Additionally it returns packages installed by a different package manager due to missing reason``. But it is still not perfect.

Reply:
Let's keep the original statement